store/tikv/snapshot.go

```diff
@@ -587,12 +587,12 @@ func extractLockFromKeyErr(keyErr *pb.KeyError) (*Lock, error) {
 }
 
 func extractKeyErr(keyErr *pb.KeyError) error {
-	failpoint.Inject("ErrMockRetryableOnly", func(val failpoint.Value) {
+	if val, err := MockRetryableErrorResp.Eval(); err == nil {
```

Comment:
Can it hurt performance?

Reply:
May cause cache miss. But in Go, i think it's not really concerned.